config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#posts", as: :user_posts
+  get "/users/:id/posts/:id", to: "users#post", as: :user_post
```

Comment:
- [ ] Great work on adding the routes 👍 but it would be nice if you add a root route as well because if you see the given wireframe then the user index page is on the root route 🤔 It's great that you have user index page on /users route but kindly add it as a root route as well to follow the project requirements 😊